challenges/functions.js

```diff
@@ -6,29 +6,47 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
```

Comment:
missing instructions
Step 1: Create a callback function
  * Create a function named consume that can take 3 parameters.
  * The first two parameteres can accept any argument
  * The last parameter accepts a callback 
  * In the body of the function return the callback with the two parameters that you created
*/